ui/app/clinical/consultation/directives/newDrugOrders.js

```diff
@@ -2,9 +2,13 @@
 
 angular.module('bahmni.clinical')
     .directive('newDrugOrders', ['messagingService', function (messagingService) {
-        var controller = function ($scope, $rootScope, appService) {
-            $scope.enableIPDFeature = appService.getAppDescriptor().getConfigValue("enableIPDFeature");
```

Comment:
@binduak @kavitha-sundararajan I can see that `enableIPDFeature` is being replaced with `allMedicinesInPrescriptionAvailableForIPD`. Was this a conscious decision, if so can you state the rationale behind it?